sys/src/cmd/gdbserver/gdbstub.c

```diff
@@ -56,13 +56,43 @@ int debug = 0;
 int attached_to_existing_pid = 0;
 Map* cormap;
 
+static struct state ks;
+
 /* support crap */
 /*
  * GDB remote protocol parser:
  */
 
 static char *hex_asc = "0123456789abcdef";
 
+char* regstrs[] = {
```

Comment:
at some point we'll want this definition to go into a file named amd64.c, and reference it in the .json as $ARCH, but that can come later.